### PR TITLE
opfiex buffs

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Oppifex smuggler pouch is now even more hidden
Adds framework to completely hide implants
Makes the organic musles for legs no longer count as robotic

Condences down and gives more pills to opifex medical backround
Opifex medical backround now gets SI bandaids, oitment, a surgery kit, some sleeping toxins and some splints
Buffs the medical opifex backround to have 5 more bio

### CODE REVIEW

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 1]
```diff
@@ -754,6 +754,27 @@
...
+	desc = "A mixed bottle of pills."
+
+
+/obj/item/storage/pill_bottle/opifex/populate_contents()
+	for(var/i in 1 to 5)
+		new /obj/item/reagent_containers/pill/bloodregen(src)
+	for(var/i in 1 to 5)
+		new /obj/item/reagent_containers/pill/prosurgeon(src)
+	for(var/i in 1 to 5)
+		new /obj/item/reagent_containers/pill/bicaridine(src)
```
Comment (benj8560) [inline code/game/objects/items/weapons/storage/firstaid.dm:769]: This is picking a random amount right? I'd much rather these much more commonly used pills where locked to 5 (if not more), Can blow through pills of brute/burn/pain meds like nothing.

Comment (Trilbyspaceclone) [reply code/game/objects/items/weapons/storage/firstaid.dm:769]: this is spawning 5 of eatch, not randomizing

**Comment by cdb-is-not-good** [comment]
im fine with buffing the medical vox, but why not just give htem ATKs and stuff? isnt the SI stuff from like....SI?

**Comment by benj8560** [comment]
> im fine with buffing the medical vox, but why not just give htem ATKs and stuff? isnt the SI stuff from like....SI?

issue with this is they cant round start with the kinda bio you need to use ATKs. You need like 40 or some shit (guessing). SI bandaids are only SLIGHTLY better then ATKs and have 0 skill needs

**Comment by benj8560** [comment]
also not like SI does not sell these bandages. Like asking why the combat tech gets a H&S weapon, :P

**Pull request merged**